### PR TITLE
[SYCLomatic] Refine the migration of cusparseHandle_t and remove "const" for blas::descriptor::get_queue()

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by tomflinda — APPROVED** [review]
LGTM

File: clang/runtime/dpct-rt/include/dpct/sparse_utils.hpp [hunk 1]
```diff
@@ -41,6 +41,17 @@ class matrix_info {
   oneapi::mkl::uplo _uplo = oneapi::mkl::uplo::upper;
   oneapi::mkl::index_base _index_base = oneapi::mkl::index_base::zero;
 };
+class descriptor {
+public:
+  descriptor() {}
+  void set_queue(queue_ptr q_ptr) noexcept { _queue_ptr = q_ptr; }
+  sycl::queue &get_queue() const noexcept { return *_queue_ptr; }
```
Comment (zhiweij1) [inline clang/runtime/dpct-rt/include/dpct/sparse_utils.hpp]:
```suggestion
  sycl::queue &get_queue() noexcept { return *_queue_ptr; }
```

Comment (zhiweij1) [reply clang/runtime/dpct-rt/include/dpct/sparse_utils.hpp]: also remove "const" in blas handle

**Review by ziranzha — APPROVED** [review]

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**